repos/websdk.proj

```diff
@@ -7,6 +7,7 @@
     <BuildCommand>$(ProjectDirectory)/build$(ShellExtension) /p:SkipTests=true</BuildCommand>
     <PackagesOutput>$(ProjectDirectory)/bin/Release/</PackagesOutput>
     <RepoApiImplemented>false</RepoApiImplemented>
+    <OrchestratedManifestBuildName>N/A</OrchestratedManifestBuildName>
```

Comment:
I'd suggest copying this comment to keep all the the info in the source so we can rip it out without question later:

https://github.com/dotnet/source-build/blob/e98bf42bf86032d0d9399e02d1c6ca80c9fa16d7/repos/corefx.proj#L13